salt/map.jinja

```diff
@@ -55,6 +55,7 @@ default:
   - python-pydot
   - python-yaml
   graph_states: False
+  log_level: info
```

Comment:
Don't you want to make metadata structure deeper?
```
log:
  level: info
```
To be able to setup syslog logging and other stuff later.